man/qc_charge_states.Rd

```diff
@@ -37,6 +37,12 @@ Calculates the charge state distribution for each sample (by count or intensity)
 }
 \examples{
 \dontrun{
-qc_charge_states(data, sample, grouping, charge_states, intensity = NULL, plot = TRUE)
+qc_charge_states(data,
```

Comment:
would do the same as above with putting the data in new line and adding spectronaut column names. since this is the example code of how to use the function.